test/lambda/mongodb/package.json

```diff
@@ -1,9 +1,10 @@
 {
-  "name": "mongodb",
+  "name": "mongodb-test-lambda",
   "version": "1.0.0",
   "description": "MongoDB Lambda Function Example",
   "main": "app.mjs",
-  "license": "MIT",
+  "private": true,
+  "license": "Apache",
```

Comment:
I don't think this is the correct format, I think I mentioned double checking the string in the driver's package file to line things up

Reply:
https://docs.npmjs.com/cli/v10/configuring-npm/package-json#license
https://spdx.org/licenses/

Reply:
Yep, looks like we need to change it to "Apache-2.0"